recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -254,7 +254,7 @@ do_install () {
     else
         # Install Vendor ICDs for OpenCL's installable client driver loader (ICDs Loader)
         install -d ${D}${sysconfdir}/OpenCL/vendors/
-        install -m 0644 ${S}/gpu-core/etc/Vivante.icd ${D}${sysconfdir}/OpenCL/vendors/Vivante.icd
+        install -m 0644 ${S}/gpu-core/etc/OpenCL/vendors/Vivante.icd ${D}${sysconfdir}/OpenCL/vendors/Vivante.icd
```

Comment:
Sorry I'm late, but there was more to this change: https://github.com/nxp-imx/meta-imx/commit/52dd61e1bc727ce4028d29b328d96f67c194a17f

Reply:
Hi @thochstein 
Fixed on this PR: #2292

Reply:
> And there's this as well. These HAL headers are removed from the package: [nxp-imx/meta-imx@ad57461](https://github.com/nxp-imx/meta-imx/commit/ad57461d9b1b57b4f195c6f3edff5b7a57dc351a)

Can you add this as well?